app/controllers/profiles_controller.ts

```diff
@@ -0,0 +1,167 @@
+import type { HttpContext } from '@adonisjs/core/http'
+import User from '#models/user'
+import Profile from '#models/profile'
+import { DateTime } from 'luxon'
+
+export default class ProfilesController {
+  // Create user profile
+  public async create({ auth, request, response }: HttpContext) {
+    try {
+      const token = auth.user?.currentAccessToken.identifier
+      if (!token) {
+        return response.badRequest({ message: 'Token not found' })
```

Comment:
Token not found is not a proper error message, please use a better user friendly message